disorder-core/src/Disorder/Core/Property.hs

```diff
@@ -65,3 +74,9 @@ p1 .^. p2 = (p1 .||. p2) .&&. neg (p1 .&&. p2)
 infixr 1 <=>
 (<=>) :: (Testable p1, Testable p2) => p1 -> p2 -> Property
 a <=> b = (a .&&. b) .||. (neg a .&&. neg b)
+
+areEquivalent :: (Eq a, Show a) => [a] -> [a] -> Property
+areEquivalent ls rs =
+  let els = ls \\ rs
+      ers = rs \\ ls
+  in counterexample ("Lists are not equivalent; extra elements: " ++ show (els,ers)) $ els ++ ers == []
```

Comment:
If we enforce the naming of the 2 lists as `actual` and `expected` then we can describe the failure message with:
```
missing:
  a, b, c

added:
  d, e, f
```